integration-sqs-to-lambda/Function.go

```diff
@@ -0,0 +1,29 @@
+package integration_sqs_to_lambda
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+func handler(ctx context.Context, event events.SQSEvent) error {
```

Comment:
Potential to remove the ctx parameter as `func handler(event T) error` is a valid handler function signature.

Reply:
refactored the handler